pkg/tidydns/tidydns_test.go

```diff
@@ -100,11 +101,16 @@ const readResponse = `{
   "brother_destination": null
 }`
 
+const userCreateResponse = `{"data":{"id":144},"status":"0"}`
+const userReadResponse = `{"modified_by":"jra-api-test","description":"Awesome test user","modified_date":"2024-12-03 14:17:22","username":"jra-test-user","auth_group":"User","name":"jra-test-user","epassword":"*****","passwd_changed_date":"2024-12-03 14:17:22","id":148,"groups":[{"groupname":"user","name":"User","notes":null,"id":2,"description":null}]}`
+const userUpdateResponse = `{"status":"0","data":{"id":146}}`
+const userDeleteResponse = `{"status":"0"}`
+
 func TestGetSubnetIDs(t *testing.T) {
 	server := httptest.NewServer(http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
 		assert.Equal(t, "10.68.0.128/26", req.URL.Query().Get("subnet"))
 		assert.Equal(t, "GET", req.Method)
-		rw.Write([]byte(subnetResponse))
+		_, _ = rw.Write([]byte(subnetResponse))
```

Comment:
I assume this is just to be verbose about ignoring return?
Not against it, just curious.

Reply:
It's explicitly ignored to silence the lint check. I don't wanna disable the lint check because it catches errors, but in this case we actually do wanna ignore the error